modules/openapi-generator/src/test/java/org/openapitools/codegen/java/JavaClientCodegenTest.java

```diff
@@ -1,1231 +0,0 @@
-/*
```

Comment:
may i know why this file is removed?

Reply:
This was temporarily removed locally during development, as this Java test single-handedly caused the tests to fail. This change was not meant to be committed, and has now been reverted to original. Apologies for this and thank you for catching.